TestFoundation/TestNSSet.swift

```diff
@@ -140,4 +144,87 @@ class TestNSSet : XCTestCase {
         }
     }
 
-}
+    func test_CountedSetBasicConstruction() {
+        let v1 = "v1".bridge()
+        let v2 = "v2".bridge()
+        let v3asv1 = "v1".bridge()
+        let set = NSCountedSet()
+        let set2 = NSCountedSet(array: [v1, v1, v2,v3asv1].bridge().bridge())
```

Comment:
the double bridge here is probably not needed since the elements in the Array are of type NSString which should fit as AnyObject.